applications/convection_diffusion_application/python_scripts/convergence_criteria_factory.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Check that applications were imported in the main script
+KratosMultiphysics.CheckRegisteredApplications("ConvectionDiffusionApplication")
+
+# Import applications
+import KratosMultiphysics.ConvectionDiffusionApplication as ConvectionDiffusionApplication
+
+# Convergence criteria class
+class convergence_criterion:
```

Comment:
isn't this file a duplicate from the one in the core?

Reply:
That-s one of the ones I was asking for, probably it was moved at some point.

Reply:
Ok I misunderstood then. Yes, this one should be in the core.